src/main/java/org/jitsi/videobridge/AbstractEndpointMessageTransport.java

```diff
@@ -109,28 +110,31 @@ private void onJSONData(
             switch (colibriClass)
             {
                 case COLIBRI_CLASS_SELECTED_ENDPOINT_CHANGED:
-                    onSelectedEndpointChangedEvent(src, jsonObject);
+                    selectedEndpointChangedEvent(src, jsonObject);
                     break;
                 case COLIBRI_CLASS_SELECTED_ENDPOINTS_CHANGED:
-                    onSelectedEndpointsChangedEvent(src, jsonObject);
+                    selectedEndpointsChangedEvent(src, jsonObject);
                     break;
                 case COLIBRI_CLASS_PINNED_ENDPOINT_CHANGED:
-                    onPinnedEndpointChangedEvent(src, jsonObject);
+                    pinnedEndpointChangedEvent(src, jsonObject);
                     break;
                 case COLIBRI_CLASS_PINNED_ENDPOINTS_CHANGED:
-                    onPinnedEndpointsChangedEvent(src, jsonObject);
+                    pinnedEndpointsChangedEvent(src, jsonObject);
                     break;
                 case COLIBRI_CLASS_CLIENT_HELLO:
-                    onClientHello(src, jsonObject);
+                    clientHello(src, jsonObject);
                     break;
                 case COLIBRI_CLASS_ENDPOINT_MESSAGE:
-                    onClientEndpointMessage(src, jsonObject);
+                    clientEndpointMessage(src, jsonObject);
                     break;
                 case COLIBRI_CLASS_LASTN_CHANGED:
-                    onLastNChangedEvent(src, jsonObject);
+                    lastNChangedEvent(src, jsonObject);
                     break;
                 case COLIBRI_CLASS_RECEIVER_VIDEO_CONSTRAINT:
-                    onReceiverVideoConstraintEvent(src, jsonObject);
+                    receiverVideoConstraintEvent(src, jsonObject);
```

Comment:
Can you add a doc somewhere (either here, in the methods that handle each message, or in the description of the constants) that describe the semantics of `RECEIVER_VIDEO_CONSTRAINT` vs `RECEIVER_VIDEO_CONSTRAINTS_CHANGED`?

Reply:
I tried adding as much details as possible here bf6d6616165cfeee6415e38db6fb2bf31d8b3775. Please let me know if further details are necessary.